Add an 'ignore' line for the HPN patch that we enable on our openssh
by default.

http://sourceforge.net/tracker/?func=detail&aid=3257504&group_id=312875&atid=1316824

diff --git a/scripts/services/sshd b/scripts/services/sshd
index a4c3354..50e8cf2 100755
--- a/scripts/services/sshd
+++ b/scripts/services/sshd
@@ -290,7 +290,8 @@ while (defined(my $ThisLine = <STDIN>)) {
        ($ThisLine =~ /^Failed keyboard-interactive for <invalid username> from/ ) or
        ($ThisLine =~ /^Keyboard-interactive \(PAM\) userauth failed/ ) or
        ($ThisLine =~ /^debug1: /) or
-       ($ThisLine =~ /Nasty PTR record .* is set up for [\da-fA-F.:]+, ignoring/)
+       ($ThisLine =~ /Nasty PTR record .* is set up for [\da-fA-F.:]+, ignoring/) or
+       ($ThisLine =~ /^SSH: Server;L[Tt]ype: /)
    ) {
       # Ignore these
    } elsif ( my ($Method,$User,$Host,$Port) = ($ThisLine =~ /^Accepted (\S+) for (\S+) from ([\d\.:a-f]+) port (\d+)/) ) {
